package.json

```diff
@@ -52,6 +52,7 @@
     "mocha": "^2.2.5",
     "mocha-jsdom": "^1.0.0",
     "react": "^0.14.0",
+    "prop-types": "^15.5.0",
```

Comment:
1. One could not use `prop-types` in his project. It should't prevent DevTools to work. React is mandatory because otherwise there will be no monitors.

3. Every example has it's own [`package.json`](https://github.com/gaearon/redux-devtools/blob/master/examples/counter/package.json) and we need dependencies to be present there as well.